settings.py

```diff
@@ -28,15 +28,6 @@
 # the maximum disjoint paths from unwanted_path
 DISJOINT_PATH_CUTOFF = 10
 
-# BATCH_INTERVAL: time interval between batch requests that will be sent to
-# flow_manager (in seconds) - zero enable sending all the requests in a row
-BATCH_INTERVAL = 0.5
-
-# BATCH_SIZE: size of a batch request that will be sent to flow_manager, in
-# number of FlowMod requests. Use 0 (zero) to disable BATCH mode, i.e. sends
-# everything at a glance
-BATCH_SIZE = 50
-
 # Default values for EVPL and EPL respectively. They are use when sb_priority
```

Comment:
@Ktmi it's cool that we'll be able to use the rate limiter soon. However check out this [thread here](https://github.com/kytos-ng/kytos/pull/453#discussion_r1490101271). The scope of this issue here is not just remove but replace. As it is here, it's just removing. `mef_eline` in prod has been fined tuned with this approximate 100 FlowMods/sec, so we need to replace accordingly. 

From what I seeing, maybe you were expecting to just configure this on kytos.conf, which make sense, but check out the linked discussion, I still see value on having the configuration on settings here and NApps themselves register/create a rate limit by default (it'd be a more gradual change), and having default configs of NApps on core it's probably not something that we should do, that's why I'm bringing this up that maybe the most gradual path here is still NApps driving the rate limit put configs/registration/creation. We need to refine this requirement/discussion.